includes/functions-html.php

```diff
@@ -757,7 +760,7 @@ function yourls_html_menu() {
 	$admin_sublinks = yourls_apply_filter( 'admin_sublinks', $admin_sublinks );
 	
 	// Now output menu
-	echo '<ul id="admin_menu">'."\n";
+	echo '<nav role="navigation"><ul id="admin_menu">'."\n";
```

Comment:
In term of HTML element, you're right. But, as I said in my general comment, I don't think you _have_ to use HTML5 elements to add ARIA roles. 
Am I wrong?